.github/workflows/ci-cd.yml

```
@@ -0,0 +1,324 @@
+    branches: [ main ]
+
+env:
+  GO_VERSION: '1.22.2'
+  NODE_VERSION: '20'
+
+jobs:
+  # Test Job
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)